integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/mailto/transport/receiver/test/MailToTransportActionAfterFailureDELETETestCase.java

```diff
@@ -37,6 +38,7 @@
 public class MailToTransportActionAfterFailureDELETETestCase extends ESBIntegrationTest {
```

Comment:
Shouldn't we extend this class from MailToTransportBaseClass. What actually sendMail method does?

Reply:
MailToTransportBaseClass is there to start and stop the green mail server. Those will be executed before and after all the test classes execute. Once the server is up, sendMail will send the email to the destination in the server.